src/main/java/com/github/manolo8/darkbot/utils/login/LoginData.java

```diff
@@ -7,19 +7,26 @@
 
 import java.lang.reflect.Type;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
 import java.util.function.BiConsumer;
+import java.util.stream.Collectors;
 
 public class LoginData {
     private static final Type PARAMS_TYPE = new TypeToken<Map<String, String>>() {}.getType();
 
+    @Getter
+    private transient final Map<Integer, LoginData> accounts = new ConcurrentHashMap<>();
```

Comment:
Avoid contaminating the LoginData class with stuff that clearly is not from a single user's login data

Instead try to pass around either a List<LoginData> or another object that is able to support 1 or many login datas and handle the logic for switching around etc.